pkg/gofr/gofr_test.go

```diff
@@ -620,57 +620,6 @@ func Test_UseMiddleware(t *testing.T) {
 	assert.Equal(t, "applied", testHeaderValue, "Test_UseMiddleware Failed! header value mismatch.")
 }
 
-// Test the UseMiddlewareWithContainer function.
```

Comment:
I disagree

As long it's here, so as long the code is not removed, the test must remain

People are using the deprecated method, and might never migrate

Reply:
@ccoVeille I completely agree that Test should remain as long as code is present.

But since we were using the `app.UseMiddlewareWithContainer` methods in test we were getting a warning that **deprecated code is still being used.**

So removed the test for the same.

Reply:
The call can be ignored with a nolint

Reply:
> @ccoVeille I completely agree that Test should remain as long as code is present.
> 
> But since we were using the `app.UseMiddlewareWithContainer` methods in test we were getting a warning that **deprecated code is still being used.**
> 
> So removed the test for the same.

So the warning was not blocking for the CI ?

Or the warning was only local to your IDE?

Reply:
Yes i checked it was local for my Goland IDE.
Did the change and ran linter, all checks passed